contrib/networkmanager-openvpn/files/sysusers.conf

```diff
@@ -0,0 +1,3 @@
+# Create networkmanager-openvpn system user
+
+u _nm_openvpn - "networkmanager-openvpn user" /var/lib/nm_openvpn /usr/bin/nologin
```

Comment:
doesn't the `/var/lib/nm_openvpn` bit need a corresponding tmpfiles entry to create it